Adafruit_Video_Looper/omxplayer.py

```diff
@@ -13,22 +15,42 @@ def __init__(self, config):
         background.
         """
         self._process = None
+        self._temp_directory = None
         self._load_config(config)
 
+    def __del__(self):
+        if self._temp_directory:
+            shutil.rmtree(self._temp_directory)
+
+    def _get_temp_directory(self):
+        if not self._temp_directory:
+            self._temp_directory = tempfile.mkdtemp()
```

Comment:
Could this cause problems with read-only file systems? or is this tempdir created in memory?

Reply:
[According to the documentation, yes, by default.](https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir) It lets the ability to use another directory with env vars.

Reply:
okay great then we just leave it like that for now.